### ISSUE TITLE
[BUG] Status isn't shown anymore on status changes

### ISSUE THREAD

**Issue by wilecoyote** [DiscordStyles/RadialStatus#30]
**Describe the bug**  
If any user changes the status, no status is shown anymore.

**To Reproduce**  
Change status.

**Screenshots**  
![image](https://user-images.githubusercontent.com/1034673/140761561-d6a77e38-0882-4af2-968d-7cfd06e8cbd3.png)
![image](https://user-images.githubusercontent.com/1034673/140761739-e525bb01-aad0-41aa-a980-83e9db86c6f9.png)

**Infomation (please complete the following information):**  
Discord channel: *All*  
OS: *Windows 10*  
Mod: *BetterDiscord*  
Discord language: *german*

**Comment by jules552** [comment]
This bug seems to appear with plugin StatusEverywhere, removing it solve the issue but a fix would be nice to have

**Comment by biast12** [comment]
@jules552 i don't have that plugin, i'm even on browser and i still got this issue

**Comment by jules552** [comment]
> @jules552 i don't have that plugin, i'm even on browser and i still got this issue

Yes, lately I experienced a similar issue without this plugin too so.. :/
